src/modules/groupMethods.ts

```diff
@@ -293,15 +293,10 @@ export const groupMethods = {
             SELECT id FROM group_hierarchy;
         `;
 
-        const subtreeTotalGroups = await prisma.membership.count({
+        const subtreeTotalGroups = await prisma.user.count({
             where: {
-                groupId: {
-                    in: treeViewGroups.map((g) => g.id),
-                },
-                archived: false,
-                user: {
-                    active: true,
-                },
+                memberships: { some: { groupId: { in: [...treeViewGroups.map((g) => g.id), id] } } },
```

Comment:
unnecessary `[]` and spread, it's already an array

Reply:
There is id added to this array to count root group members